eng/ci/build-core-tools-host-artifacts-windows.yml

```diff
@@ -1,18 +1,7 @@
 trigger:
   branches:
     include:
-    - feature/oop-host
-  paths:
-    include:
-      - ./host/src/**
-
-pr:
-  branches:
-    include:
-    - feature/oop-host
-  paths:
-    include:
-      - ./host/src/**
+    - feature/*
```

Comment:
yeah I thought so too but when I had the previous changes the pipeline wasn't triggering automatically which is why I removed this to see if it would work. Any suggestions for why the custom host build wasn't trigerred?

Reply:
I wonder if this might just be a syntax issue on the path. Wildcards seem to be limited to a single asterisk from this doc, but I don't know the double asterisk is defined elsewhere: https://learn.microsoft.com/en-us/azure/devops/pipelines/repos/azure-repos-git?view=azure-devops&tabs=yaml#wildcards

I'm also not sure about using `feature/*` here. We really only need this for `feature/oop-host`, so we should leave this with the full name, I think.

Reply:
Also should we include an explicit `pr: none` similar to the official build?

Reply:
Ok I removed all `**` so hopefully the build will get triggered here. I've instead added all the possible paths that we can modify to have the build trigger.

I've also only restricted this to `feature/oop-host` and added an explicit `pr: none`